source/docs/software/telemetry/persistent-alerts.rst

```diff
@@ -0,0 +1,57 @@
+# Recording Faults with Persistent Alerts
+
+Robots encounter a variety of fault conditions: disconnected sensors/motors, invalid mechanism states, initialization failures, etc. While the FRC Driver Station provides a :ref:`console interface <docs/software/driverstation/driver-station:Messages Tab>` for instantaneous alerts, the risk of missing important messages makes it poorly suited to communicate faults that persist over time.
+
+Instead, the ``Alert`` class ([Java](https://github.wpilib.org/allwpilib/docs/development/java/edu/wpi/first/wpilibj/Alert.html), [C++](https://github.wpilib.org/allwpilib/docs/development/cpp/classfrc_1_1_alert.html), [Python](https://robotpy.readthedocs.io/projects/robotpy/en/stable/wpilib/Alert.html#wpilib.Alert)) can be used for managing persistent alerts published via Network Tables. Alerts are assigned a priority (*error*, *warning*, or *info*) and can be *activated* or *deactivated* in robot code. The set of active alerts can be displayed on a dashboard, as shown below in Shuffleboard.
+
+.. image:: images/alerts.png
+   :alt: A screenshot of the alerts widget in Shuffleboard, with several active alerts.
+
+Active alerts are automatically displayed in order based on priority and how recently they were activated, with newer and more critical alerts at the top of the list. This provides a mechanism for drivers (on the field) and programmers (off the field) to quickly assess the state of the robot and determine whether it is in a "match-ready" state. Highly descriptive alerts also allow drive teams to adapt to faults in real time, such as switching strategies in case of a subsystem failure.
+
+## API Usage
+
+Alerts are created and managed using the ``Alert`` class. Typically, an alert should be instantiated once and stored as a class member. The ``set`` method is used to activate or deactivate the alert. Change detection is built-in, so calling ``set`` multiple times with the same value does affect the order in which alerts are displayed.
```

Comment:
Nowhere is it mentioned that set takes a boolean, Especially without any code in the examples, I think that's important.